neo4j_utils/_driver.py

```diff
@@ -834,7 +834,10 @@ def db_status(
 
         with self.fallback():
 
-            resp, summary = self.query(query=query, db=name)
+            try:
```

Comment:
I think this error handling shouldn't happen here, the `query` method already takes care of it, doesn't it?